Gopkg.toml

```diff
@@ -70,18 +70,6 @@
   name = "github.com/Azure/go-autorest"
   version = "11.1.2"
 
-[[constraint]]
```

Comment:
(general comment, not related to this line) it looked like running `dep ensure` again would drop some additional packages - specifically saw some additional Azure-related packages being removed when I ran it

Reply:
Hmyes, I'm fighting with dep, trying to figure it out.

Reply:
I tried this again - was able to simply run `dep ensure` after checking out your branch and have the extra packages removed.